tests/cross/src/test/scala/tests/hover/HoverNamedArgSuite.scala

```diff
@@ -29,10 +29,6 @@ class HoverNamedArgSuite extends BaseHoverSuite {
         """|```scala
```

Comment:
Would probably be great to show the contents of `@param` for named, but this can be an improvement for the future. Curiously, why didn't the Scala 2 behaviour change?

Reply:
In Scala 2 we don't handle named arguments currently and we show hover for the method instead